utils/tag_parsers.py

```diff
@@ -38,36 +38,47 @@ def medium_check(get_data, form_type) -> str:
 
 
 # getting Open graph tags data
-def open_graph(request_object, get_data) -> dict:
+def open_graph(tupl) -> dict:
+    request_object = tupl[0]
+
+    get_data = {

```

Comment:
use `defaultdict` instead adding all these entries upfront

Reply:
Okay, I'll do that 👍